mod-circulation/src/main/resources/vega/mod-circulation/features/requests.feature

```diff
@@ -752,4 +752,41 @@ Feature: Requests tests
     And match $.requests[0].id == extRequestId
 
   Scenario: Run tlr-request feature
-    * call read('classpath:vega/mod-circulation/features/tlr-requests.feature')
\ No newline at end of file
+    * call read('classpath:vega/mod-circulation/features/tlr-requests.feature')
+
+  Scenario: VuFind integration - backward compatibility. Requests shouldn't have instanceId, holdingRecordId, requestLevel fields as mandatory
+    * def extItemId = call uuid1
+    * def extUserId = call uuid1
+    * def extItemBarcode = 'FAT-2178IBC'
+    * def extUserBarcode = 'FAT-2178UBC'
+    * def extUserId = call uuid1
+    * def extRequestType = 'Page'
+
+    # post an item
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostItem') { extItemId: #(extItemId), extItemBarcode: #(extItemBarcode) }
+
+    # post an user
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostUser') { extUserId: #(extUserId), extUserBarcode: #(extUserBarcode), extGroupId: #(fourthUserGroupId) }
```

Comment:
Can we just pass values for extItemBarcode and extUserBarcode?, instead of defining extra variables because they are just used once here